packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -83,12 +83,25 @@ export default class AutomateProvider extends GenericProvider {
           percyBuildUrl: this.buildInfo.url,
           state: 'begin'
         });
+        // Selenium Hub, set status error Code to 13 if an error is thrown
+        // Handling error with Selenium dialect is != W3C
+        if (result?.status === 13) throw new Error(result?.value || 'Got invalid error response');
```

Comment:
`result.value` will contain the error message from SeleniumHub correct ?

Reply:
Correct